test/contrib/test_module.py

```diff
@@ -7,7 +7,8 @@
 from numpy.testing import assert_allclose
 import pytest
 
-from jax import random, test_util
+from jax import random, tree_map
```

Comment:
nit: it is better to import `tree_map` from `tree_util` (see [this comment](https://github.com/google/jax/blob/main/jax/__init__.py#L104))